src/compiler/build/compiler-ctx.ts

```diff
@@ -17,7 +17,7 @@ export class CompilerContext implements d.CompilerCtx {
   activeFilesUpdated: string[] = [];
   activeDirsAdded: string[] = [];
   activeDirsDeleted: string[] = [];
-  addWatchDir: (path: string) => void = noop;
+  addWatchDir: (path: string, recursive: boolean) => void = noop;
```

Comment:
Found this bug while refactoring this. Apparently this was typed incorrectly, and downstream the signature didn't expect a second parameter. That's because the interface definition was divorced from the class definition. There may be more that will pop up this way too.